### PR TITLE
Adds release-it functionality

### PR DESCRIPTION
**Pull request by juliarosenson**
This PR adds release-it configuartion so that we can use the CLI tool to cut releases

### CODE REVIEW

**Comment by rr-codes** [comment]
make sure to also publish each version to NPM as well :)

File: README.md [hunk 1]
```diff
@@ -43,6 +43,13 @@ jobs:
           notion-db: ${{ secrets.NOTION_DATABASE }}
 ```
 
+## Using `release-it`
+
+1. Locally, on `master` (make sure it's up to date), execute `GITHUB_TOKEN=<TOKEN> release-it`. (Alternatively, set `GITHUB_TOKEN` as a system environment variable)
+2. Follow the interactive prompts, selecting `Yes` for all options.
+3. When selecting the increment, choose `patch` when the release is only bug fixes. For new features, choose `minor`. For major changes, choose `major`.
+
+Release-It will then automatically generate a GitHub release with the changelog inside.
```
Comment (juliarosenson) [inline README.md:52]: Since this is a public repo, do we want to remove this info from the README?

Comment (marissamarym) [reply README.md:52]: I think it's ok to keep it in, since our tokens are still secret and only maintainers will be able to do this

**Review by marissamarym — APPROVED** [review]
LGTM

**Comment by rr-codes** [comment]
> make sure to also publish each version to NPM as well :)

@juliarosenson whoops, I was mistaking this with our NPM packages `mack` and `martian`; this doesn't need to be published to NPM at all since it's just a GH action

**Pull request merged**